lib/tasks/stats.rake

```diff
@@ -604,4 +604,39 @@ namespace :stats do
 
     package.serialize 'eval_queries.xlsx'
   end
+
+  task all_time_twilio_stats: :environment do
+    twilio_stats_since_date(Date.new(2020,3,1))
+  end
+
+  task month_twilio_stats: :environment do
+    twilio_stats_since_date(1.month.ago.to_date)
+  end
+
+  def twilio_stats_since_date(since_date)
+    client = Twilio::REST::Client.new(ENV['TWILLIO_API_ACCOUNT'], ENV['TWILLIO_API_KEY'])
+    sms_total = (client.usage.records.list(category: 'sms', start_date: since_date).collect do |record| record.count end)[0]
```

Comment:
would it make sense to factor out this logic into a method like:

```
def twilio_stat(category, since_date)
    (client.usage.records.list(category: category, start_date: since_date).collect do |record| record.count end)[0]&.reverse&.gsub(/...(?=.)/,'\&,')&.reverse
end
```

and then call it as:
```
puts "SMS Total: #{twilio_stat('sms', since_date)}"
```